install/installer/cmd/testdata/render/aws-setup/output.golden

```diff
@@ -3276,6 +3276,20 @@ data:
               }
             ]
           },
+          {
+            "name": "gitpod_vscode_web_load_total",
+            "help": "Total count of supervisor frontend client loading / errored windows",
```

Comment:
help should talk about VS Code Web Workbench, i.e. `Total count of attempts to load VS Code Web workbench` cc @mustard-mh